src/components/search.js

```diff
@@ -1,11 +1,32 @@
 import React from 'react'
 import emojiIndex from '../utils/emoji-index'
 
+function rtrim(str,s) {
+    if (s == undefined)
+        s = '\\s';
+    return str.replace(new RegExp("[" + s + "]*$"), '');
+}
+function ltrim(str,s) {
+    if (s == undefined)
+        s = '\\s';
+    return str.replace(new RegExp("^[" + s + "]*"), '');
+}
+
 export default class Search extends React.Component {
+  constructor(props){
+    super(props);
+    if(props.newdata){
+      emojiIndex.updateData(this.props.newdata);
+    }
+    this.state = {
+      placeholder : null
+    };
+  }
   handleChange() {
     var { input } = this.refs,
-        value = input.value
+        value = input.value;
 
+    value = ltrim(rtrim(value,':'),':');
```

Comment:
privously wasnt able to search starting with `:`  so added as default functionality. removed `:` from both sides